lib/dbservice/colleges/college.ex

```diff
@@ -0,0 +1,106 @@
+defmodule Dbservice.Colleges.College do
+  @moduledoc """
+  Schema for managing college information.
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+  import Ecto.Query, warn: false
+  alias Dbservice.Repo
+
+  @required_fields [
+    :college_id,
+    :institute
+  ]
+
+  @optional_fields [
+    :state,
+    :place,
+    :dist_code,
```

Comment:
see `school` table once.. it says district code... 

even `place` sounds a bit weird. see `user` table.. what is it being called

even `co_ed` ... its `gender_type` in school table. 

i just want to maintain consistency

pritam suggested `type` instead of `college_type` .. what are your thoughts on this?